frontend/src/metabase/visualizations/PieChart.jsx

```diff
@@ -100,7 +100,7 @@ export default class PieChart extends Component {
         }
 
         let legendTitles = slices.map(slice => [
-            slice.key === "Other" ? slice.key : formatDimension(slice.key, false),
+            slice.key === "Other" ? slice.key : formatDimension(slice.key, true),
```

Comment:
No but stuff got shuffled around when I took the entity name stuff out so there might be some random things in here